marketing/views.py

```diff
@@ -138,46 +141,33 @@ def post(self, request):
     def get(self, request):
         data_count = self.querysets.count()
         params = request.query_params.dict()
+        querysets = self.querysets
+
 
         if params.get("id"):
             querysets = self.querysets.filter(id=params.get("id"))
-            jobs = JobGetSerializer(querysets, many=True).data
-            return Response(
-                {"data": jobs, "total_count": data_count}, status=status.HTTP_200_OK
```

Comment:
Please specify why this is removed.

Reply:
This is removed because pagination is being applied and it is using these lines functionality, so I have called that function from utils/pagination at last after checking all the params such as id, location, name. We want pagination to be applied on every GET request of Jobs whether it be fetching using id, name, or other parameter.
Also, calling same function separately would lead to redundancy in the code.

If you want I could change it by calling that pagination function everytime in these 'if' conditions.